tracer/build/_build/docker/gitlab/install_net35.ps1

```diff
@@ -3,98 +3,13 @@
 #
 $ErrorActionPreference = "Stop"
 
-$UpgradeTable = @{
-    1809 = @{
-    ## Taken from the mcr.microsoft.com/dotnet/framework/runtime:3.5 Dockerfile:
-    ## https://github.com/microsoft/dotnet-framework-docker/blob/26597e42d157cc1e09d1e0dc8f23c32e6c3d1467/3.5/runtime/windowsservercore-ltsc2019/Dockerfile
+# Enable Windows Update service
+Set-Service -Name wuauserv -StartupType Automatic
+Start-Service -Name wuauserv
 
-        netfxzip = "https://dotnetbinaries.blob.core.windows.net/dockerassets/microsoft-windows-netfx3-1809.zip";
-        netfxsha256 = "439f0b98438d9d302a21765cdfe2f28af784aa6a63dfff11c4e1a8a20e9fdf93"
-        # Trying to get the patch with HTTPS yields:
-        # 2022-02-24T15:53:44.6586746+00:00: curl: (60) schannel: SNI or certificate check failed: SEC_E_WRONG_PRINCIPAL (0x80090322) - The target principal name is incorrect.
-        patch = "http://download.windowsupdate.com/c/msdownload/update/software/updt/2020/01/windows10.0-kb4534119-x64_a2dce2c83c58ea57145e9069f403d4a5d4f98713.msu";
-        patchsha256 = "0287ba4106ba5462ba542e0124c4211f50bbce1123167375f125ea27b8f48632"
-        expandedpatch = "windows10.0-kb4534119-x64.cab"
-    };
-    1909 = @{
-    ## Taken from the mcr.microsoft.com/dotnet/framework/runtime:3.5 Dockerfile:
-    ## https://github.com/microsoft/dotnet-framework-docker/blob/abc2ca65b28058f7c71ec8cd8763a8fbf2a9c03f/3.5/runtime/windowsservercore-1909/Dockerfile
+# Install .NET Framework 3.5
```

Comment:
.NET Framework 3.5 ?? 😅

Reply:
WIX 🙁

Reply:
Let's add "update WIX" to our list of R&D week projects 😄